lms/templates/student_account/login_and_register.html

```diff
@@ -1,6 +1,7 @@
 <%!
```

Comment:
I realized `paver run_quality` just generates the reports. I needed to run `paver run_eslint` before. Hopefully it'll pass this time.